src/multiversioning.zig

```diff
@@ -748,13 +754,11 @@ pub const Multiversion = struct {
             allocator.free(std.mem.span(self.args_envp.args[0].?));
             allocator.free(self.args_envp.args);
         }
-
-        self.timeout_start_enabled = false;
```

Comment:
Do we need to `self.timeout.stop()`?

Reply:
We don't, but I've `self.* = undefined` for good measure to make sure this isn't used after deinit by mistake.